### ISSUE TITLE
Knockback stunlock

### ISSUE THREAD

**Issue by NullCGT** [k21971/EvilHack#14]
While I think it's awesome that you're adding SpliceHack's monster knockback mechanic, you will **definitely** want to take a look at https://github.com/NullCGT/SpliceHack/issues/15. The way monster clobbering is implemented in EvilHack right now, your're going to have a lot of players angry about getting stun-locked against walls. You will also want to make sure that monsters with additional attacks after a clobber do not crash the game by attempting an attack after the target has been knocked away. I'm not sure whether that's an issue, but it's certainly worth checking.

Additionally, you should be aware of the following:
- Hurtling into a wall will cause additional damage.
- Hurtling into a monster makes that monster angry. This was discovered in SpliceHack when a player was punched into One-Eyed Sam by a giant, and proceeded to get beheaded.
- Due to the presence of lava and fire giants, this change makes the valkyrie quest insanely difficult without levitation. I've done it, but it was incredibly nerve-wracking.

Basically, you took a feature that I already considered absurdly, cartoonishly evil and made it even nastier. I think that's the best endorsement I can give EvilHack :)

**Comment by k21971** [comment]
Thanks for letting me know! I'll have a look at it.

**Comment by k21971** [comment]
Because of the decrease in hurtle distance between SpliceHack's implementation of this and EvilHack's, I'm not seeing any stun lock from knockback at all. I've ran over 100 separate knockback hits from a monster with AD_CLOB and I'm not witnessing any issues - walls, doors, anythings. Am I missing something?

**Comment by k21971** [comment]
Closing this, at least for now. Testing does not reveal any kind of stun lock when clobbered back into a wall.
